static/app/components/events/interfaces/spans/spanProfileDetails.tsx

```diff
@@ -30,24 +30,26 @@ import useOrganization from 'sentry/utils/useOrganization';
 import useProjects from 'sentry/utils/useProjects';
 import {useProfileGroup} from 'sentry/views/profiling/profileGroupProvider';
 
-import type {SpanType} from './types';
-
 const MAX_STACK_DEPTH = 8;
 const MAX_TOP_NODES = 5;
 const MIN_TOP_NODES = 3;
 const TOP_NODE_MIN_COUNT = 3;
 
-interface SpanProfileDetailsProps {
+export interface SpanProfileDetailsProps {
   event: Readonly<EventTransaction>;
-  span: Readonly<SpanType>;
+  span: Readonly<{
+    end_timestamp: number;
+    span_id: string;
+    start_timestamp: number;
+  }>;
```

Comment:
Are these underscores here to match the backend response? Typically we do camel case in js